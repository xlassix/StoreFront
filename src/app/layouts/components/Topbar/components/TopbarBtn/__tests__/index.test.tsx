import * as React from 'react';
import { render } from '@testing-library/react';

import { TopbarBtn } from '..';

describe('<TopbarBtn  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <TopbarBtn type="button" onClick={jest.fn}>
        <div>Child compoent</div>
      </TopbarBtn>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
