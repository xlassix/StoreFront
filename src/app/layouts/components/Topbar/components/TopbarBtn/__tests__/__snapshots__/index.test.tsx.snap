// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopbarBtn  /> should match snapshot 1`] = `
.c0 {
  width: 60px;
  height: 60px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background: transparent;
  border: none;
  cursor: pointer;
  -webkit-transition: 0.3s;
  transition: 0.3s;
}

.c0:focus {
  outline: none;
}

@media (min-width:768px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

@media (max-width:576px) {
  .c0 {
    margin-left: 220px;
    margin-bottom: 15px;
  }
}

@media (max-width:320px) {
  .c0 {
    margin-left: 140px;
  }
}

<button
  class="c0"
/>
`;
